Detectors/ITSMFT/MFT/reconstruction/src/devices/FileSink.cxx

```diff
@@ -101,7 +101,7 @@ void FileSink::Init()
   
   for ( mNObjects = 0 ; mNObjects < mBranchNames.size() ; mNObjects++ ) {
 
-    LOG(INFO) << "FileSink::Init >>>>> Creating output branch \"" << mClassNames[mNObjects] << "\" with name \"" << mBranchNames[mNObjects] << "\"";
+    LOG(INFO) << R"(FileSink::Init >>>>> Creating output branch ")" << mClassNames[mNObjects] << R"(" with name ")" << mBranchNames[mNObjects] << R"(")";
 
```

Comment:
I am not sure about those changes. They are not "visually" attractive and are harder to read. What do others think?